src/main/java/com/igridin/desktop_calculator/ClassicCalcOperations.java

```diff
@@ -1,3 +1,4 @@
+package com.igridin.desktop_calculator;
```

Comment:
As the sources are now moved to maven-recommended package structure, this needs to be specified.